### DISCUSSION TITLE
For example; FAQ 1 Typo?

### DISCUSSION THREAD

**Discussion by WapackDev** [mpdifran/Memento-Issues#69] [Ideas]
First [FAQ](https://www.getmemento.ca/faq.html):
> Q: Why are there already reminders in the app when I installed it?
> A: **_Reminder_** uses the same database as the default Reminders app, so you can edit and view your reminders in either. That means any reminders you add will automatically be synced through iCloud to your other devices, and reminders added by Siri will appear in the app. You can even delete the default Reminders app and just use **_Reminder_** if you want.

I'm guessing you meant **Memento**.

**Comment by mpdifran** [comment]
I've fixed the copy [here](https://www.getmemento.ca/faq.html), thanks for spotting this!
